ui-admin/src/user/DeleteUserModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { Store } from 'react-notifications-component'
+import Api, { AdminUser, Portal } from 'api/api'
+import { failureNotification, successNotification } from 'util/notifications'
+
+/** Modal to remove a user from a portal. */
+const DeleteUserModal = ({ subjUser, portal, userDeleted, onDismiss }:
+                           {subjUser: AdminUser, portal: Portal, userDeleted: () => void, onDismiss: () => void}) => {
+  const [usernameConfirm, setUsernameConfirm] = useState('')
+  const removeString = `remove ${subjUser.username}`
+  const canRemove = usernameConfirm === removeString
+
+  const doRemove = async () => {
+    try {
+      await Api.removePortalUser(subjUser, portal.shortcode)
+      Store.addNotification(successNotification(`${subjUser.username} removed`))
+      userDeleted()
+      onDismiss()
+    } catch (e) {
+      console.log('error removing user', e)
+      Store.addNotification(failureNotification('Error removing user'))
+    }
```

Comment:
There's a relatively new (and not-particularly well-named) Api helper function we've added that standardizes the error reporting to the user.  
```suggestion
      await doApiLoad(() => {
         Api.removePortalUser(subjUser, portal.shortcode)
         Store.addNotification(successNotification(`${subjUser.username} removed`))
         userDeleted()
         onDismiss()
    }, {customErrorMsg: 'Error removing user'} )
```